### ISSUE TITLE
Duplicate camera files for rebel line

### ISSUE THREAD

**Issue by 51systems** [pedrocr/rawloader#19]
The rebel line of canon cameras (T*i) have duplicate camera definition files - one for the official model and another for the localized model name.

I don't think the localized model name file is necessary as the camera is populated from the alias section of the official model file.

**Comment by pedrocr** [comment]
Yep seems like that's the case. Probably a leftover from before I implemented aliases. I doubt it happens in many cases. Probably a good thing to fix after I implement the regression testing setup I've thought about. That way I can do the changes and check that the output is still all the same.
